addons/mail/static/src/components/composer/composer.xml

```diff
@@ -111,7 +112,7 @@
                                         <i class="fa fa-smile-o"/>
                                     </button>
                                     <t t-set="opened">
-                                        <EmojisPopover t-ref="emojisPopover" onEmojiSelection="_onEmojiSelection"/>
+                                        <EmojisPopover ref="'emojisPopover'" refs="refs" onEmojiSelection="_onEmojiSelection"/>
```

Comment:
make a separate PR to introduce emojiPopoverView model if necessary (if I'm seeing correctly it's now the only component missing its view model in the diff of this PR)

Reply:
PR with use of popover view containing emoji list: https://github.com/odoo/odoo/pull/81877